app/src/main/java/com/jjswigut/matters/ui/MatterListFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.jjswigut.matters.ui
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.jjswigut.matters.ui.MatterListRecyclerViewAdapter
+import com.jjswigut.matters.R
+import com.jjswigut.matters.dummy.DummyContent
+
+/**
+ * A fragment representing a list of Items.
+ */
+class MatterListFragment : Fragment() {
+
+    private var columnCount = 1
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_matter_list, container, false)
+
+        // Set the adapter
+        if (view is RecyclerView) {
```

Comment:
is this ever true? it looks like the root view inside of `fragent_matter_list` is a RelativeLayout so i don't think this would ever be true

Reply:
I'm gonna try this another way. Again, I don't really understand how to do this is in the most simple way because every sample I look at seems to have an entirely different way of setting things up

Reply:
completely understandable! let me know and we can go over how i'd do it as well - but only if ya want to!

Reply:
I'd love to know how you would do it! I think I figured out a fairly simple way in my recent PR but  you let me know!